src/components/deploymentConfig/DeploymentTemplateView.tsx

```diff
@@ -771,18 +771,8 @@ export const DeploymentTemplateEditorView = ({
         setBasicFieldValues(_basicFieldValues)
     }
 
-    return yamlMode || (selectedChart.name !== ROLLOUT_DEPLOYMENT && selectedChart?.name !== DEPLOYMENT) ? (
-        <>
-            {showReadme && (
-                <div className="dt-readme dc__border-right">
-                    <div className="code-editor__header flex left fs-12 fw-6 cn-9">Readme</div>
-                    {chartConfigLoading ? (
-                        <Progressing pageLoader />
-                    ) : (
-                        <MarkDown markdown={readme} className="dt-readme-markdown" />
-                    )}
-                </div>
-            )}
+    const displayCodeEditor = (): JSX.Element => {
```

Comment:
Please rename it to renderCodeEditor or something related with the prefix render while rendering JSX element